tests/nodeos_high_transaction_test.py

```diff
@@ -0,0 +1,379 @@
+                    "last_trans_index" : transOrder - 1,
+                    "last_bnum" : lastBlockNum,
+                })
+                if newestBlockNum > lastBlockNum:
+                    missingTransactions[-1]["highest_block_seen"] = newestBlockNum
+            blockNum = Node.getTransBlockNum(trans)
+            assert blockNum is not None, Print("ERROR: could not retrieve block num from transId: %s, trans: %s" % (transId, json.dumps(trans, indent=2)))
+        else:
+            block = transToBlock[transId]
+            blockNum = block["block_num"]
+            assert blockNum is not None, Print("ERROR: could not retrieve block num for block retrieved for transId: %s, block: %s" % (transId, json.dumps(block, indent=2)))
+
+        if lastBlockNum is not None:
+            if blockNum > lastBlockNum + transBlocksBehind or blockNum + transBlocksBehind < lastBlockNum:
+                transBlockOrderWeird.append({
+                    "newer_trans_id" : transId,
+                    "newer_trans_index" : transOrder,
+                    "newer_bnum" : blockNum,
+                    "last_trans_id" : lastTransId,
+                    "last_trans_index" : transOrder - 1,
+                    "last_bnum" : lastBlockNum
+                })
+                if newestBlockNum > lastBlockNum:
+                    last = transBlockOrderWeird[-1]
+                    last["older_trans_id"] = newestBlockNumTransId
+                    last["older_trans_index"] = newestBlockNumTransOrder
+                    last["older_bnum"] = newestBlockNum
+
+        if newestBlockNum is None:
+            newestBlockNum = blockNum
+            newestBlockNumTransId = transId
+            newestBlockNumTransOrder = transOrder
+        elif blockNum > newestBlockNum:
+            newestBlockNum = blockNum
+            newestBlockNumTransId = transId
+            newestBlockNumTransOrder = transOrder
+
+        lastTransId = transId
+        transOrder += 1
+        lastBlockNum = blockNum
+
+    nextTime = time.perf_counter()
+    Print("Validating transfers took %s sec" % (nextTime - startTranferValidationTime))
+
+    delayedReportError = False
+    if len(missingTransactions) > 0:
+        verboseOutput = "Missing transaction information: [" if Utils.Debug else "Missing transaction ids: ["
+        first = True
+        for missingTrans in missingTransactions:
+            if not first:
+                verboseOutput += ", "
+            verboseOutput += missingTrans if Utils.Debug else missingTrans["newer_trans_id"]
+            first = False
+
+        verboseOutput += "]"
+        Utils.Print("ERROR: There are %d missing transactions.  %s" % (len(missingTransactions), verboseOutput))
+        delayedReportError = True
+
+    if len(transBlockOrderWeird) > 0:
+        verboseOutput = "Delayed transaction information: [" if Utils.Debug else "Delayed transaction ids: ["
+        first = True
+        for trans in transBlockOrderWeird:
```

Comment:
As above, string join of list comprehension.

Reply:
ok